api/protobuf/buf.gen.yaml

```diff
@@ -0,0 +1,25 @@
+version: v1
+managed:
+  enabled: true
+  go_package_prefix:
+    # todo: re-organize this, keeping it to the old reference so consumers don't break
+    default: github.com/onflow/archive/api/archive
+    except:
+      - buf.build/googleapis/googleapis
+      - buf.build/srikrsna/protoc-gen-gotag
+
+plugins:
+  # renovate: datasource=github-releases depName=protocolbuffers/protobuf-go
+  - remote: buf.build/protocolbuffers/plugins/go:v1.28.1-1
+    out: ../archive
+    opt:
+      - paths=source_relative
+
+  # Faster protobuf implementation for gRPC.
+  # See: https://github.com/planetscale/vtprotobuf
+  #
+  # # renovate: datasource=github-releases depName=planetscale/vtprotobuf
+  - remote: buf.build/planetscale/plugins/vtproto:v0.3.0
+    out: ../archive
+    opt:
+      - paths=source_relative,features=marshal+unmarshal+size+pool+grpc
```

Comment:
I do not see the `FromVTPool` in the generated proto, so you might need to also add the following options:
https://github.com/vitessio/vitess/blob/371be9e919adc7c3c277834974fb666656775f5d/Makefile#L277-L279



cc: https://github.com/planetscale/vtprotobuf/issues/33